frontend/src/metabase/App.tsx

```diff
@@ -50,6 +52,7 @@ const getErrorComponent = ({ status, data, context }: AppErrorDescriptor) => {
 interface AppStateProps {
   errorPage: AppErrorDescriptor | null;
   isAdminApp: boolean;
+  bannerMessageDescriptor?: string;
```

Comment:
Please remove it here and add a "container" component for `AppBanner` and use `mapStateToProps` there directly

Reply:
To not render the banner, you would need to return `null` from the component itself.